pkg/scheduler/objects/utilities_test.go

```diff
@@ -35,25 +35,27 @@ const (
 	appID2  = "app-2"
 	aKey    = "alloc-1"
 	nodeID1 = "node-1"
+	zero    = uint64(0)
 )
 
 // Create the root queue, base for all testing
 func createRootQueue(maxRes map[string]string) (*Queue, error) {
-	return createManagedQueueWithProps(nil, "root", true, maxRes, nil)
+	return createManagedQueueWithProps(nil, "root", true, maxRes, nil, zero)
 }
 
 // wrapper around the create call using the one syntax for all queue types
-func createManagedQueue(parentSQ *Queue, name string, parent bool, maxRes map[string]string) (*Queue, error) {
-	return createManagedQueueWithProps(parentSQ, name, parent, maxRes, nil)
+func createManagedQueue(parentSQ *Queue, name string, parent bool, maxRes map[string]string, maxApps uint64) (*Queue, error) {
+	return createManagedQueueWithProps(parentSQ, name, parent, maxRes, nil, maxApps)
 }
 
 // create managed queue with props set
-func createManagedQueueWithProps(parentSQ *Queue, name string, parent bool, maxRes, props map[string]string) (*Queue, error) {
+func createManagedQueueWithProps(parentSQ *Queue, name string, parent bool, maxRes, props map[string]string, maxApps uint64) (*Queue, error) {
```

Comment:
Instead of changing this signature add a new function that allows setting max apps and leave this as is. i.e. `createManagedQueuePropsMaxApps(....)`
Move the real code there and make `createManagedQueueWithProps()` call that one